gb-clock/src/tick.rs

```diff
@@ -0,0 +1,17 @@
+#[derive(PartialEq, Eq)]
+/// Identify a type of a tick.
+pub enum Tick {
+    /// a CPU tick, executing once per clock cycle.
+    TCycle,
+    /// a PPU tick, executing 4 times per clock cycle.
+    MCycle,
```

Comment:
It should be `TCycle` for `PPU` and `MCycle` for `CPU` ?

Reply:
Yes, fixed it